### PR TITLE
Replace cocoapods with regular framework

### PR DESCRIPTION
**Pull request by AlexeyTsvetkov**

### CODE REVIEW

**Comment by SebastianAigner** [comment]
Assuming this is awaiting the release according to the [version number](https://github.com/JetBrains/compose-multiplatform-ios-android-template/pull/19/files#diff-3d103fc7c312a3e136f88e81cef592424b8af2464c468116545c4d22d6edcf19R25)  :)

**Review by pjBooms — CHANGES_REQUESTED** [review]
Please update it to 1.5.0-beta01. I think we can merge it then, taking into account that iOS is in alpha

**Comment by StylianosGakis** [comment]
Could this PR also include a change in the README which points to the right documentation about how KMP developers can start working with this vs using cocoapods?
A lot of us are used to be doing things 1 way, so it would be nice to learn how to work with frameworks (which as I understand will be the suggested way going forward) and understand why changes like these https://github.com/JetBrains/compose-multiplatform-ios-android-template/pull/19/files#diff-3128a88752afa7f03b453c0c30cbc563d3998b87fc54a1267bd1f1c943341d87R327-R333 are necessary and how to do them ourselves (even if it's from inside Xcode) if we need to make our own thing in the future.

**Comment by AlexeyTsvetkov** [comment]
@StylianosGakis you don't have to migrate to regular frameworks. Cocoapods will continue to work just fine. But if you don't use cocoapods as a package manager, then using non-cocoapods frameworks is one less dependency to set up for every team member. We decided to change our examples and templates for that reason: cocoapods is only needed for pulling existing obj-c packages, which is not what most new users would do. If you want to do the same, there is an instruction in the following PR https://github.com/JetBrains/compose-multiplatform/pull/3340 (it's not very obvious place, maybe we should copy it somewhere else)

**Comment by StylianosGakis** [comment]
Ah thank you so much, yes this is exactly what I was looking for 🤩
I think it would be very valuable to exist in some official documentation too, especially if as I said this becomes part of the template that people copy, which means that they will no matter what be exposed to using frameworks.
Thanks a lot for the reply btw 😊

**Comment by ColtonIdle** [comment]
Just came here to say that when i see cocoapods i think outdated. so yeah itd be nice to have a starting point where you can use spm.

**Review by gianfigueroa — APPROVED** [review]

File: gradle.properties [hunk 1]
```diff
@@ -22,4 +22,4 @@ android.minSdk=24
 #Versions
 kotlin.version=1.8.20
 agp.version=7.4.2
-compose.version=1.4.1
```
Comment (gianfigueroa) [inline gradle.properties:25]: Update version

**Review by pjBooms — APPROVED** [review]

**Review by SebastianAigner — APPROVED** [review]

**Pull request merged**